Tools/install.sh

```diff
@@ -99,12 +99,34 @@ elif [ -n "$(command -v dnf)" ]; then
     DIST=fedora
     PKG_INSTALL="sudo dnf install -y"
 else
+    _OK=1
```

Comment:
`OK` is a bit terse. `TOOLS_MISSING` would be better.
Please don't use _ idenfiers - follow the style of existing script.

Reply:
I will rework this.